tools/traffic/table/blob_metadata.go

```diff
@@ -0,0 +1,69 @@
+package table
+
+import "errors"
+
+// BlobMetadata encapsulates various information about a blob written by the traffic generator.
+type BlobMetadata struct {
+	// key of the blob, set when the blob is initially uploaded.
+	key []byte
+
+	// checksum of the blob.
+	checksum *[16]byte
+
+	// size of the blob, in bytes.
+	size uint
+
+	// blobIndex of the blob.
+	blobIndex uint
```

Comment:
What is blob index?

Reply:
Yea, this index is the the position of the blob in the batch (the batch is a list of blobs)